client/src/components/Content.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import Bundle from "./bundle/bundle";
```

Comment:
This import is broken. Did you mean to import `./bundle/Bundle`?

Reply:
Resolved